src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -77,7 +80,13 @@ public class ProjectMapper : SnapshotMapperBase<ModelProject, SnapshotProject>
 
       public override async Task<ModelProject> MapToModel(SnapshotProject snapshot)
       {
-         var project = new ModelProject {Description = snapshot.Description};
+         var project = new ModelProject
```

Comment:
Creating project from Snapshot. We update the CreationMetaData to use the version that was saved in Snapshot